packages/instant/src/style/fonts.ts

```diff
@@ -1,10 +1,10 @@
-import { injectGlobal } from './theme';
-
 export const fonts = {
     include: () => {
         // Inject the inter-ui font into the page
-        return injectGlobal`
-            @import url('https://rsms.me/inter/inter-ui.css');
-        `;
+        const head = document.head || document.getElementsByTagName('head')[0];
```

Comment:
do you think it's possible that an integrator's page may not have a `head` tag?  i guess its unlikely

Reply:
This is a good point.